packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -203,12 +203,13 @@ export default Component.extend({
         columns = $.extend(true, [], get(this, 'options.columns'));
 
     return columns.map( column => {
-      let sort =  arr(sorts).findBy('metric', canonicalizeMetric(column.field)) || {};
-      let sortDirection;
+      let {field, type} = column,
+          sort =  arr(sorts).findBy('metric', type === 'dateTime' ? type : canonicalizeMetric(field)) || {},
```

Comment:
Can we break this down some more? 
Extract ` type === 'dateTime' ? type : canonicalizeMetric(field)` into a different var maybe?
I feel there is too much happening in this line and a little hard to follow.

Reply:
Fixed